AUTHORS

```diff
@@ -112,6 +112,7 @@ Li Yi                                   denverdino@gmail.com
 Maciej Kwiek                            maciej@isovalent.com
 Maciej Skrocki                          maciejskrocki@google.com
 Madhu Challa                            madhu@cilium.io
+MaiReo                                  sawako.saki@gmail.com
```

Comment:
Would be good to get the proper full-name here and add to .mailmap.

/cc @MaiReo